checkpoint.go

```diff
@@ -40,13 +42,14 @@ checkpointed.`,
 		}
 		defer destroy(container)
 		options := criuOptions(context)
+		preDump := context.Bool("pre-dump")
```

Comment:
technically, pre-dump is not a criu-option, it is a subcmd like dump(checkpoint), so I take this as an outer arg.
But, I can change this if people want checkpoint clear : )